BUILDING.md

```diff
@@ -35,6 +34,6 @@ machine while running with the Linux target (simulation mode). Obviously you
 need an SGX-capable machine to try out the SGX target. There are multiple ways
 which you might procure an SGX-capable machine, including but not limited to:
 - buy or build an Intel NUC (TODO: include specific model number)
-- use a `DCsv2`-series VM on Azure (https://aka.ms/accgetstarted)
+- use a [`DCsv2`-series (from DC1s_v2 up to DC8s_V2) VM on Azure](https://aka.ms/accgetstarted)
```

Comment:
nit: DC8_v2 does not have a "s" in its name

Reply:
Nice catch. Corrected.